datajoint/external.py

```diff
@@ -330,7 +330,7 @@ def _need_checksum(local_filepath):
                         )
             if not _need_checksum(local_filepath):
                 logger.warning(
-                    f"WARNING SKIPPED CHECKSUM FOR FILE WITH HASH: {contents_hash}"
+                    f"Warning skipped checksum for file with hash: {contents_hash}"
```

Comment:
would it be more valuable to log the filepath than the checksum?

Reply:
It might be best to log both, that way if they have any problems they can manually verify the checksum.

Reply:
```suggestion
                    f"Skipped checksum for file with hash: {contents_hash}"
```
Already at the [WARNING] level with the logger.

Reply:
We should at least check for matching file size if we're not checking for checksum. Expected file size can be fetched from external table